docs/src/main/asciidoc/mongodb-panache-guide.adoc

```diff
@@ -0,0 +1,505 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Simplified MongoDB with Panache
+
+include::./attributes.adoc[]
+:config-file: application.properties
+
+MongoDB is a well known NoSQL Database that is widely used, but using it's raw API can be cumberson as you need to express your entities and your queries in MongoDB Document.
+
+MongoDB with Panache will provides active record style entities (and repositories) like you have in Hibernate ORM with Panache and focuses on making your entities trivial and fun to write in Quarkus.
+
+== First: an example
+
+What we're doing in Panache is allow you to write your MongoDB entities like this:
+
+[source,java]
+--
+public class Person extends PanacheMongoEntity {
+    public String name;
+    public LocalDate birth;
+    public Status status;
+
+    public static Person findByName(String name){
+      return find("name", name).firstResult();
+    }
+
+    public static List<Person> findAlive(){
+      return list("status", Status.Alive);
+    }
+
+    public static void deleteStefs(){
+      delete("name", "Stef");
+    }
+}
+--
+
+You have noticed how much more compact and readable the code is?
+Does this look interesting? Read on!
+
+NOTE: the `list()` method might be surprising at first. It takes fragments of PancheQL queries (subset of JPQL) and contextualize the rest. That makes for very concise but yet readable code. Native MongoDB queries are also supported.
```

Comment:
/s/PancheQL/PanacheQL

Reply:
/s/That makes for very concise/That makes for very a concise

Reply:
are you sure for the second one? I copied it from Hibernate with Panache guide ...

Reply:
first one done